test/integration/setting_js_test.rb

```diff
@@ -29,4 +29,78 @@ class SettingJSTest < IntegrationTestWithJavascript
     assert_index_page(settings_path, "Settings", false, true, false)
     assert page.has_link?("My Pretty Setting Label", :href => "#category_label_test_settings_tab")
   end
+
+  test "general headers values" do
+    visit settings_path
+
+    expect page.has_content?('Name')
+    expect page.has_content?('Value')
+    expect page.has_content?('Description')
+  end
+
+  test "edit single arrays values" do
+    visit settings_path
+    find('button#http_proxy_except_list').click
+
+    input = find('textarea#setting-textarea-http_proxy_except_list', visible: true)
+    expect input.value.empty?
+
+    input.fill_in with: 'testArr'
+    find('button[data-ouia-component-id="submit-edit-btn"]').click
+    expect page.has_content?('updated setting')
+
+    expect page.has_content?('[testArr]')
+  end
+
+  test "edit empty array values" do
+    visit settings_path
+    find('button#http_proxy_except_list').click
+
+    input = find('textarea#setting-textarea-http_proxy_except_list', visible: true)
+    expect input.value.empty?
+
+    input.fill_in with: ''
+    find('button[data-ouia-component-id="submit-edit-btn"]').click
+    expect page.has_content?('updated setting')
+
+    expect page.has_content?('[]')
+  end
+
+  test "edit multi array values" do
+    visit settings_path
+    find('button#http_proxy_except_list').click
```

Comment:
I think this should work:
```suggestion
    click_button '#http_proxy_except_list'
```